.gitignore

```diff
@@ -60,3 +60,4 @@ build/mage_output_file.go
 !**/**/testdata/*.rego
 
 release-notes.md
+config/*.*.bk
```

Comment:
i modify `config/local.yml` but dont want to check in my changes, so this allows me to rename my changed local.yml to local.yml.bk so I can save without commiting